packages/config/config/devices/0x010f/fgd212_3.5.json

```
@@ -97,16 +99,22 @@
 			"$import": "../templates/master_template.json#state_after_power_failure_off_prev"
 		},
 		"10": {
-			"label": "Timer functionality (auto - off)",
-			"description": "Defines the setting of the timer",
+			"label": "Timer Functionality (auto - off)",
+			"description": "Will automatically turn off after the specified time from switching on",
 			"valueSize": 2,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 32767,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		"11": {
-			"label": "ALL ON/ALL OFF function",
-			"description": "The parameter specifies the available functions.",
+			"label": "ALL ON/ALL OFF Function",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 65535,
```

I must have missed that on the last review. This should be 
```suggestion
			"maxValue": 255,
```
since the option values don't go above 255. Or is the value size 2 a mistake?

---

The valueSize 2 is correct. Changed maxValue to 255.